src/components/NewMessageModal.vue

```diff
@@ -294,6 +294,51 @@ export default {
 		},
 		toHtml,
 		plain,
+		async onNewSentMailbox(data, account) {
+			showWarning(t('mail', 'Setting Sent default folder'))
+			let newSentMailboxId = null
+			const mailboxes = this.mainStore.getMailboxes(data.accountId)
+			const sentMailboxId = mailboxes.find((mailbox) => mailbox.name === account.personalNamespace + 'Sent' || mailbox.name === account.personalNamespace + t('mail', 'Sent'))?.databaseId
+			if (sentMailboxId) {
+				await this.setSentMailboxAndResend(account, sentMailboxId, data)
```

Comment:
I'm missing a return or anything else that leaves this method after setting the existing mailbox

Reply:
> hamza221 marked this conversation as resolved.

Isn't this precisely the reason why I'm running into errors despite being able to send the message?

To have constructive and comprehensive collaboration please be more elaborative. If you close my comment as resolved silently I assume it's because you have addressed it.